static/app/components/dropdownMenu/list.tsx

```diff
@@ -57,6 +57,10 @@ export interface DropdownMenuListProps
    * Whether the menu should close when an item has been clicked/selected
    */
   closeOnSelect?: boolean;
+  /**
+   * To be displayed below the menu items
+   */
+  menuFooter?: React.ReactChild;
```

Comment:
Needed to add this to DropdownMenu in order to add the info at the bottom. It already accepts `menuTitle` so this is similar.